actions/terra-k8s-linter/policy/deployments/deny.rego

```diff
@@ -0,0 +1,32 @@
+package main
+
+import data.kubernetes
+
+
+name = input.metadata.name
+
+required_deployment_labels {
+  input.metadata.labels["app.kubernetes.io/name"]
+  input.metadata.labels["app.kubernetes.io/instance"]
+  input.metadata.labels["helm.sh/chart"]
+  input.metadata.labels["app.kubernetes.io/managed-by"]
+}
+
+deny_required_deployment_labels[msg] {
+  input.kind == "Deployment"
+  not required_deployment_labels
+  msg = sprintf("%s must include Kubernetes recommended labels", [name])
+}
+
+deny_replicas[msg] {
+  input.kind == "Deployment"
+  input.metadata.annotations["bio.terra.linter/replicas_exception"] != "enabled"
+  input.spec.replicas < 3
+  msg := sprintf("Must have at least 3 replicas. %s has %d replicas.", [name, input.spec.replicas])
+}
+
+deny_revision_history[msg] {
+  input.kind == "Deployment"
+  input.spec.revisionHistoryLimit != 0
```

Comment:
same here about the hardcoded/static numbers thing

Reply:
This is defined in the README. Each action is required to have it.